src/python/pants/backend/python/lint/ruff/skip_field.py

```diff
@@ -11,17 +11,31 @@
 from pants.engine.target import BoolField
 
 
-class SkipRuffField(BoolField):
-    alias = "skip_ruff"
+class SkipRuffCheckField(BoolField):
+    alias = "skip_ruff_check"
     default = False
-    help = "If true, don't run ruff on this target's code."
+    help = "If true, don't run the ruff checker on this target's code."
+
+    deprecated_alias = "skip_ruff"
+    deprecated_alias_removal_version = "2.23.0.dev0"
+
+
+class SkipRuffFormatField(BoolField):
+    alias = "skip_ruff_format"
```

Comment:
It's a bit annoying to do and definitely in the realm of fine-tuning, but I wonder if it'd be better to split these into their respective backends. In particular, I believe this current arrangement means that someone only using the `...ruff.check` backend would still have a `skip_ruff_format` attached to all the Python targets, and vice versa.

Reply:
I took a crack at splitting this out - main thing that makes me nervous about this is the number of files/rules moved around; I'm not sure if some rule will get omitted somewhere that causes a regression.